editor.planx.uk/src/@planx/components/ExternalPortal/Editor.tsx

```diff
@@ -51,20 +155,78 @@ const ExternalPortalForm: React.FC<{
             flow that it references.
           </span>
         </ModalSectionContent>
-        <ModalSectionContent title="Pick a flow">
-          <select
+        <ModalSectionContent
+          key={"team-section"}
+          title="Select a team (optional)"
+        >
+          <SelectMultiple
```

Comment:
There's an odd interaction here - if I have a flow selected, and then change the team the fist flow of this team gets selected which is quite unexpected.